Code/Framework/AzToolsFramework/Tests/MetadataManagerTests.cpp

```diff
@@ -310,7 +310,7 @@ namespace UnitTest
         EXPECT_TRUE(m_metadata->GetValueVersion("mockfile", "/Test", version));
         EXPECT_EQ(version, 1);
 
-        rapidjson_ly::Value inValue;
+        rapidjson_ly::Document inValue;
         EXPECT_TRUE(m_metadata->GetJsonValue("mockfile", "/Test", inValue));
```

Comment:
This should probably use `ASSERT_TRUE`. Continuing to access members of the document if `GetJsonValue` failed will probably return fail conditions, and the verifications done further in this test will surely fail.